### ISSUE TITLE
Trouble connecting to server

### ISSUE THREAD

**Issue by beltxd** [SmileycorpMC/Deeper-Depths#4]
https://mclo.gs/7xmuc29 

Seems to be a conflicting mod or file of some sorts.

**Comment by aking4125** [comment]
Can confirm the above error. Cannot join server: 
![image](https://github.com/user-attachments/assets/3b1f24f4-738f-49bb-81c4-50645c4b59e7)

[latest.log](https://github.com/user-attachments/files/17344100/latest.log)

**Comment by beltxd** [comment]
> Can confirm the above error. Cannot join server: ![image](https://private-user-images.githubusercontent.com/11141831/375794021-3b1f24f4-738f-49bb-81c4-50645c4b59e7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg4OTg3ODEsIm5iZiI6MTcyODg5ODQ4MSwicGF0aCI6Ii8xMTE0MTgzMS8zNzU3OTQwMjEtM2IxZjI0ZjQtNzM4Zi00OWJiLTgxYzQtNTA2NDVjNGI1OWU3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDE0VDA5MzQ0MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTVlYmM1NTE0MWY3ZjkxOWYwNzI0ODUxNTUzY2U2YTEwZGVmYWQxNjc0MWQ4YTI1OWQ1NGM5OWM5MTA0YTM2MmUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.8lvxbGuj9gMtOMSpbkIvGp2SRgChvO0NgwVXQnyuIho)
> 
> [latest.log](https://github.com/user-attachments/files/17344100/latest.log)

I looked at the code for the recipe registry and changed a few things up and now it works. I can send the changes if you want. Can serve as a temporary solution before switching to jsons

**Comment by SmileycorpMC** [comment]
> > Can confirm the above error. Cannot join server: ![image](https://private-user-images.githubusercontent.com/11141831/375794021-3b1f24f4-738f-49bb-81c4-50645c4b59e7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg4OTg3ODEsIm5iZiI6MTcyODg5ODQ4MSwicGF0aCI6Ii8xMTE0MTgzMS8zNzU3OTQwMjEtM2IxZjI0ZjQtNzM4Zi00OWJiLTgxYzQtNTA2NDVjNGI1OWU3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDE0VDA5MzQ0MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTVlYmM1NTE0MWY3ZjkxOWYwNzI0ODUxNTUzY2U2YTEwZGVmYWQxNjc0MWQ4YTI1OWQ1NGM5OWM5MTA0YTM2MmUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.8lvxbGuj9gMtOMSpbkIvGp2SRgChvO0NgwVXQnyuIho)
> > [latest.log](https://github.com/user-attachments/files/17344100/latest.log)
> 
> I looked at the code for the recipe registry and changed a few things up and now it works. I can send the changes if you want. Can serve as a temporary solution before switching to jsons in the future possibly

What was the change you made that fixed it?

I'd rather not switch over to json recipes if possible, we have 151 recipes to handle some which I'd like to be configurable.

**Comment by beltxd** [comment]
line 37 registerCraftingRecipes(); was removed under public static void register recipes event.

Replace private static void registerCraftingRecipes() { with public static void registerRecipes(RegistryEvent.Register<IRecipe> event) {  in line 71. 

Bit of a weird fix, but it should work in servers. Worked in my modpack fine.

**Comment by beltxd** [comment]
> > > Can confirm the above error. Cannot join server: ![image](https://private-user-images.githubusercontent.com/11141831/375794021-3b1f24f4-738f-49bb-81c4-50645c4b59e7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg4OTg3ODEsIm5iZiI6MTcyODg5ODQ4MSwicGF0aCI6Ii8xMTE0MTgzMS8zNzU3OTQwMjEtM2IxZjI0ZjQtNzM4Zi00OWJiLTgxYzQtNTA2NDVjNGI1OWU3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDE0VDA5MzQ0MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTVlYmM1NTE0MWY3ZjkxOWYwNzI0ODUxNTUzY2U2YTEwZGVmYWQxNjc0MWQ4YTI1OWQ1NGM5OWM5MTA0YTM2MmUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.8lvxbGuj9gMtOMSpbkIvGp2SRgChvO0NgwVXQnyuIho)
> > > [latest.log](https://github.com/user-attachments/files/17344100/latest.log)
> > 
> > 
> > I looked at the code for the recipe registry and changed a few things up and now it works. I can send the changes if you want. Can serve as a temporary solution before switching to jsons in the future possibly
> 
> What was the change you made that fixed it?
> 
> I'd rather not switch over to json recipes if possible, we have 151 recipes to handle some which I'd like to be configurable.

line 37 registerCraftingRecipes(); was removed under public static void register recipes event.

Replace private static void registerCraftingRecipes() { with public static void registerRecipes(RegistryEvent.Register event) {
in line 71.

 Bit of a weird fix, but it should work in servers. Worked in my modpack fine.

**Comment by beltxd** [comment]
![image](https://github.com/user-attachments/assets/ad8a5336-9417-4c79-a7ce-6360fcc93180)

**Comment by beltxd** [comment]
![image](https://github.com/user-attachments/assets/5b181104-0457-4774-8368-dc5b575682ec)

my bad for deleting that initial change (if you saw it) accidentally wrote the wrong thing, fixed it up again

**Comment by SmileycorpMC** [comment]
Wait I think I know why it's erroring, I think it's using the wrong event. If you try the commit I just pushed, does it work?

**Comment by beltxd** [comment]
Yep, it works. Thanks for the changes. I saw the event initially and thought something was up so I did a little digging around and ended up with my version but it wasn't polished, it was more of a haste option.
